libp2p/netMessenger_test.go

```diff
@@ -239,6 +245,15 @@ func TestNewNetworkMessenger_NilSyncTimerShouldErr(t *testing.T) {
 	assert.True(t, errors.Is(err, p2p.ErrNilSyncTimer))
 }
 
+func TestNewNetworkMessenger_NilP2pPrivateKey(t *testing.T) {
```

Comment:
missing unitests for `args.P2pSingleSigner` and `args.P2pKeyGenerator`

Reply:
done